suites/reef/rgw/tier-2_rgw_ms-archive.yaml

```diff
@@ -407,3 +407,54 @@ tests:
       polarion-id: CEPH-83573390
       module: sanity_rgw_multisite.py
       name: test multipart+encrypt object access at the archive site
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            set-env: true
```

Comment:
@viduship do we need set-env in all these tests?

Reply:
no we do not

Reply:
removed